templates/index.html

```diff
@@ -767,6 +767,21 @@ <h3> <b>Line Follower</b> </h3>
                                                 </div>
                                             </div>
                                         </div>
+
+                                        <!-- Help: Debug Tab -->
+                                        <div class="card">
+                                            <div class="card-header" id="help-debug-heading">
+                                                <details class="btn btn-link btn-block text-left" type="button" data-toggle="collapse" data-target="#help-debug" aria-expanded="true" aria-controls="help-debug">
+                                                    <summary>Feedback/Bug Reports</summary>
+                                                </details>
+                                            </div>
+                                            <div id="help-debug" class="collapse" aria-labelledby="help-debug">
+                                                <div class="card-body">
+                                                    If you encountered a bug that you feel requires attention from staff, please submit a report at this link:
+                                                    <a href="https://forms.gle/i4q48q7fxBxRxDSb6" target="_blank"> Submit a Bug Report </a>
+                                                </div>
+                                            </div>
+                                        </div>
```

Comment:
It looks like this was also added in by accident while resolving a merge conflict. It should be deleted.